InventoryInStorePickup/Model/SearchRequest/Area/SearchTerm/CityParser.php

```diff
@@ -36,9 +38,26 @@ public function __construct(DelimiterParser $parser)
     public function execute(string $searchTerm, DataObject $dataObject): void
     {
         if (empty($dataObject->getData(self::POSTCODE))) {
-            $dataObject->setData(self::CITY, $this->parser->getSearchQuery($searchTerm));
+            $dataObject->setData(self::CITY, $this->getSearchQuery($searchTerm));
         } else {
             $dataObject->setData(self::CITY, '');
         }
     }
+
+    /**
+     * Parse and return search query from search term.
+     *
+     * @param string $searchTerm
+     *
+     * @return string
+     */
+    public function getSearchQuery(string $searchTerm): string
```

Comment:
Are you sure this method should be public?

Reply:
Public -> private.